src/Microsoft.Health.Fhir.Core/Features/Search/ContinuationTokenConverter.cs

```diff
@@ -26,7 +44,15 @@ public static string Decode(string encodedContinuationToken)
         public static string Encode(string continuationToken)
         {
             EnsureArg.IsNotEmptyOrWhiteSpace(continuationToken);
-            return Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(continuationToken));
+
+            var buffer = System.Text.Encoding.UTF8.GetBytes(continuationToken);
```

Comment:
Do we need any exception handling here?  I am not sure if creating or writing to the DeflateStream can possibly throw an exception

Reply:
Seems fine to let those bubble up, which as you saw would be unlikely.